heltour/tournament/views.py

```diff
@@ -616,20 +616,24 @@ def view(self, username):
         pairings = player.pairings.exclude(scheduled_time=None)
         return self.ical_from_pairings_list(pairings, calendar_title, uid_component)
 
-class RegisterView(LeagueView):
+class RegisterView(LoginRequiredMixin, LeagueView):
+
```

Comment:
The register view should also have some protection against duplicate registrations. This will prevent things like double clicking from resulting in two registrations.  We can discuss the best way to prevent these.